backend/src/main/java/com/google/backend/servlets/AlertsDataServlet.java

```diff
@@ -48,4 +59,30 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.setContentType("application/json;");
     response.getWriter().println(new Gson().toJson(alertList));
   }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) 
+  throws ServletException, IOException {
+    // Read from the request body, which contains data in the format "alertId statusToChangeTo".
+    // For example, the body could be "4785074604081152 RESOLVED".
+    BufferedReader reader = request.getReader();
```

Comment:
v nit: maybe refactor the lines that are in charge of extracting reader in to a separate function, then maybe the longer comment above can be omitted